torch/onnx/symbolic.py

```diff
@@ -534,6 +534,8 @@ def RNN_symbolic_builder(cell_type, *args, **kwargs):
         return LSTM_symbolic_builder(*args, **kwargs)
     elif cell_type == 'GRU':
         return GRU_symbolic_builder(*args, **kwargs)
+    elif cell_type.startswith('RNN_'):
```

Comment:
Here the meaning of "RNN_" should be the in-place version. Probably you should handle both RNN_ and RNN.